tests/ceph_installer/test_cephadm.py

```diff
@@ -104,7 +104,7 @@ def run(ceph_cluster: Ceph, **kwargs) -> int:
     try:
         config = kwargs["config"]
         cephadm = CephAdmin(cluster=ceph_cluster, **config)
-        steps = config.get("steps")
+        steps = config.get("steps", [])
```

Comment:
Are there any scenarios where `steps` wouldn't be used?

If there is none, can we make it mandatory i.e. `steps = config[steps]`

Reply:
Currently from this PR, `verify_health` is one which doesn't need `steps`.